CP/Assignment1-STL/Ankit_Jain/pair.cpp.txt

```diff
@@ -0,0 +1,35 @@
+#include<bits/stdc++.h>
+using namespace std;
+bool myfunc(pair<int,int> &a, pair<int,int> &b) // I didn't actually get this concept.
+{
+return a.second<b.second;
```

Comment:
If second is equal check for first

Reply:
I dont know how to change them to .cpp

Reply:
Make a new file (pair.cpp not pair.cpp.txt) and put your code there, delete these old ones

Reply:
there is an option for changing the file using online editor. shall I change the file name using it? will it work